clang/test/Headers/Inputs/include/stdint.h

```diff
@@ -16,4 +16,12 @@ typedef unsigned __INTPTR_TYPE__ uintptr_t;
 #error Every target should have __INTPTR_TYPE__
 #endif
 
+#ifdef __INTPTR_MAX__
+#define  INTPTR_MAX    __INTPTR_MAX__
+#endif
+
+#ifdef __UINTPTR_MAX__
+#define UINTPTR_MAX   __UINTPTR_MAX__
+#endif
+
```

Comment:
Are these changes required to make this work?

Reply:
Alternatively we could just delete this `stdint.h` altogether, because this only happens because it's using this one instead. But I assume that we override the regular `stdint.h` to reduce the number of live definitions in tests or something.

Reply:
This header is useful when invoking front end directly such that we don't need to pass things like `-internal-isystem` which usually are added by compiler driver.